datahub-web-react/src/app/shared/tags/DomainLink.tsx

```diff
@@ -10,12 +10,10 @@ import { ANTD_GRAY } from '../../entity/shared/constants';
 
 const DomainLinkContainer = styled(Link)`
     display: inline-block;
-    margin-bottom: 8px;
 `;
 
 const DomainWrapper = styled.span`
     display: inline-block;
-    margin-bottom: 8px;
```

Comment:
I believe this DomainLink (and the DataProductLink and tags below) are all used in the sidebar on profile pages. do things still look good there with these CSS changes?

Reply:
yeah I think the spacing may be a little off in that sidebar. you can always wrap the components like `DomainLink` in a new spacer span styled component with `margin-bottom:8px;` for that specific use case! probably better there anyways than forcing styles everywhere it's used.